assets/grants/marco-bolo-grant.json

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": "https://schema.org",
+  "@type": "MonetaryGrant",
+  "@id": "",
+  "name": "Grant Agreement No. 101082021 (MARCO-BOLO)"
+  "identifier": [
+    "101082021",
+    "10068180 (MS)",
+    "10063994 (MBA)",
+    "10048178 (NOC)"
+   ],
+    "funder":
+      "@id": "LINK-TO-EU-JSON",
```

Comment:
Isn't there Horizon match funding from UKRI as well?

Reply:
@dblear @kmexter is there information published online about the UK side funding ?